tests/manager/manager-executeWriteCommand_error-002.phpt

```diff
@@ -21,13 +21,73 @@ try {
     $manager->executeWriteCommand(DATABASE_NAME, $command, ['writeConcern' => new MongoDB\Driver\WriteConcern("undefined")]);
 } catch (MongoDB\Driver\Exception\CommandException $e) {
     printf("%s(%d): %s\n", get_class($e), $e->getCode(), $e->getMessage());
-    assert($e->getMessage() === "Write Concern error: " . $e->getResultDocument()->writeConcernError->errmsg);
-    assert($e->getCode() === $e->getResultDocument()->writeConcernError->code);
+    var_dump($e->getResultDocument());
 }
 
 ?>
 ===DONE===
 <?php exit(0); ?>
---EXPECT--
+--EXPECTF--
 MongoDB\Driver\Exception\CommandException(79): Write Concern error: No write concern mode named 'undefined' found in replica set configuration
+object(stdClass)#13 (6) {
```

Comment:
The object identifiers should always use `%d` in `EXPECTF` output, since these can change. The property counter ("(6)" in this case) can usually be left as-is, but I'd also suggest replacing it with `(%d)` so that we can use a wildcard for some extra fields here.